Makefile

```
@@ -28,7 +28,7 @@ verifiers: getdeps fmt lint ruleguard check-gen
 
 check-gen:
 	@go generate ./... >/dev/null
-	@git diff --exit-code >/dev/null || echo "Non-committed changes detected, please commit them to proceed."
+	@(! git diff --name-only | grep '_gen.go$$') || (echo "Non-committed changes in auto-generated code is detected, please commit them to proceed." && false)
```

```suggestion
	@(! git diff --name-only | grep '_gen.go$$') || (echo "Non-committed changes in auto-generated code is detected, please commit them to proceed." && false)
```

It is okay to complain about non-committed files as well not just autogenerated @vadmeste

---

> It is okay to complain about non-committed files as well not just autogenerated @vadmeste

Actually it can be annoying, because downloading verifies, such golang ci lint, etc.. can update go.mod and go.sum during CI

---

They shouldn't update go.mod or go.sum - @vadmeste for such deps we should download GO111MODULE=off